CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # USGS Iridium Short Burst Data (SBD) Decoder Library
 
+## 2.2.0 - 2022-01-04
```

Comment:
Probably should be today's date but I'm cool with leaving it

Reply:
I use the date of the last commit